provider/nas/nas.go

```diff
@@ -298,30 +297,28 @@ func (p *NasPlugin) checkOptions(opt *NasOptions) error {
 	}
 	defer conn.Close()
 
-	// nfs version, support 4.0
+	// default nfs version 4.0
 	if opt.Vers == "" {
 		opt.Vers = "4.0"
 	}
-	// only vers=3 is supported by nfs
-	if strings.HasPrefix(opt.Vers, "3") {
+
+	// set vers=3.0 to vers=3, because vers=3.0 mount cmd submit error "parsing error on 'vers=' option"
+	if strings.HasPrefix(opt.Vers, "3.0") {
 		opt.Vers = "3"
 	}
-	nfsV4 := false
+
+	// if input vers=4, then set vers=4.0
 	if strings.HasPrefix(opt.Vers, "4") {
-		nfsV4 = true
+		opt.Vers = "4.0"
 	}
 
 	// nfs server path
 	if opt.Path == "" {
-		if nfsV4 {
-			opt.Path = defaultV4Path
-		} else {
-			opt.Path = defaultV3Path
-		}
+		opt.Path = defaultV3V4Path
 	}
-	if !strings.HasPrefix(opt.Path, "/") {
-		log.Errorf("NAS: Path should be empty or start with /, %s", opt.Path)
-		return errors.New("NAS: Path should be empty or start with /: " + opt.Path)
+	if !strings.HasPrefix(opt.Path, "/nfsshare") {
+		log.Errorf("NAS: Path should start with /nfsshare, %s", opt.Path)
+		return errors.New("NAS: Path should start with /nfsshare, %s: " + opt.Path)
```

Comment:
format string is not supported, in `errors.New()`.  should use `+` as before, or use `fmt.Errorf()` instead